### PR TITLE
Add html coverage of extension to atoumtask

### PR DESCRIPTION
**Pull request by oxman**
Add HTML Coverage of Report Extension to AtoumTask for Phing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # `dev-master`
 
 * [#612](https://github.com/atoum/atoum/pull/612) Add telemetry support to AtoumTask ([@oxman])
+* [#620](https://github.com/atoum/atoum/pull/620) Add HTML Coverage of Report Extension to AtoumTask for Phing ([@oxman])
```
Comment (Hywan) [inline CHANGELOG.md]: Please, place 620 before 612 :-). We append items.

File: tests/units/resources/phing/AtoumTask.php [hunk 2]
```diff
@@ -548,6 +552,30 @@ public function testSetCodeCoverageCloverPath()
...
 
+		public function testGetSetCodeCoverageReportExtensionPath()
+		{
+			$this
+				->given($task = new testedClass())
+				->then
+				->object($task->setCodeCoverageReportExtensionPath($path = uniqid()))->isIdenticalTo($task)
+				->string($task->getCodeCoverageReportExtensionPath())->isEqualTo($path)
+				->object($task->setCodeCoverageReportExtensionPath($path = rand(1, PHP_INT_MAX)))->isIdenticalTo($task)
+				->string($task->getCodeCoverageReportExtensionPath())->isEqualTo((string) $path)
```
Comment (mikaelrandy) [inline tests/units/resources/phing/AtoumTask.php]: Maybe respect the same indentation as other test methods

File: tests/units/resources/phing/AtoumTask.php [hunk 3]
```diff
@@ -548,6 +552,30 @@ public function testSetCodeCoverageCloverPath()
...
+
+		public function testGetSetCodeCoverageReportExtensionUrl()
+		{
+			$this
+				->given($task = new testedClass())
+				->then
+				->object($task->setCodeCoverageReportExtensionUrl($path = uniqid()))->isIdenticalTo($task)
+				->string($task->getCodeCoverageReportExtensionUrl())->isEqualTo($path)
+				->object($task->setCodeCoverageReportExtensionUrl($path = rand(1, PHP_INT_MAX)))->isIdenticalTo($task)
+				->string($task->getCodeCoverageReportExtensionUrl())->isEqualTo((string) $path)
```
Comment (mikaelrandy) [inline tests/units/resources/phing/AtoumTask.php]: same here

**Comment by mikaelrandy** [comment]
👍

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 # `dev-master`
 
+* [#620](https://github.com/atoum/atoum/pull/620) Add HTML Coverage of Report Extension to AtoumTask for Phing ([@oxman])
```
Comment (jubianchi) [inline CHANGELOG.md]: "Add HTML coverage report from [reports extension](https://github.com/atoum/reports-extension)"

**Comment by jubianchi** [comment]
minor change in the readme and then it will be merged :)

Thanks!

**Pull request merged**